agent/xds/testdata/routes/connect-proxy-with-chain-and-splitter.latest.golden

```diff
@@ -12,13 +12,45 @@
           ],
           "name": "db",
           "routes": [
+            {
+              "match": {
+                "prefix": "/big-side"
+              },
+              "route": {
+                "cluster": "big-side.default.dc1.internal.11111111-2222-3333-4444-555555555555.consul",
+                "idleTimeout": "0s",
+                "timeout": "0s"
```

Comment:
This shows the request and idle timeouts being disabled, from the service router configured in this test case